packages/swift/build.sh

```diff
@@ -77,9 +79,6 @@ termux_step_post_get_source() {
 			$TERMUX_PKG_CACHEDIR/$SWIFT_BIN.tar.gz \
 			86b849d9f6ba2eda4e12ea5eafaa0748bffcd6272466b514c2b0fd4a829c63a4
 	fi
-
-	# The Swift compiler searches for the clang headers so symlink against them.
-	export TERMUX_CLANG_VERSION=$(grep ^TERMUX_PKG_VERSION= $TERMUX_PKG_BUILDER_DIR/../libllvm/build.sh | cut -f2 -d=)
```

Comment:
This is used by [the Swift compiler config later](https://github.com/termux/termux-packages/blob/master/packages/swift/swift-cmake.patch#L88), so still needed.